src/main/java/carpetfixes/mixins/reIntroduced/PlayerManager_LlamaRidingDupeMixin_VMPCompat.java

```diff
@@ -0,0 +1,42 @@
+package carpetfixes.mixins.reIntroduced;
+
+import carpetfixes.CFSettings;
+import carpetfixes.CarpetFixesServer;
+import carpetfixes.helpers.Utils;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityType;
+import net.minecraft.nbt.NbtCompound;
+import net.minecraft.server.PlayerManager;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.Nullable;
+import org.spongepowered.asm.mixin.Dynamic;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import java.util.function.Function;
+
```

Comment:
Add a restriction so vmp must be present for this to run. Just cleans up the checks a bit more without the errors

Reply:
I'll change it ;)